src/app/feature-flag/notification-footer/feature-footer.component.less

```diff
@@ -0,0 +1,73 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.feature-footer {
+  position:absolute;
+  bottom:0;
+  width:100%;
+  height:60px;   /* Height of the footer */
+  background: transparent;
+}
+
+
+.round-button(@color-for-feature) {
+  display:block;
+  width:50px;
+  height:50px;
+  line-height:50px;
+  border-radius: 50%;
+  color: white;
+  text-align:center;
+  background: @color-for-feature;
+  box-shadow: 0 0 3px gray;
+  font-size:20px;
+  font-weight:bold;
+  float:right;
+  margin-right: 8%;
+  &:hover {
+    background: darken((@color-for-feature), 10%);
+  }
+}
+
+f8-feature-footer .tooltip-inner {
+  background: @color-pf-black-150;
+  color: @color-pf-black-600;
+  border-color: @color-pf-black-700;
+}
+
+f8-feature-footer .tooltip-arrow {
+  display: none;
+}
+
+
+f8-feature-footer .arrow {
+  display: none;
+}
+
+.empty {
+  .round-button(@color-pf-black-400;);
+}
+
+// INTERNAL
+.internal {
+  .round-button(@color-internal);
+}
+
+// EXPERIMENTAL
+.experimental {
+  .round-button(@color-experimental);
+}
+
+// BETA
+.beta {
+  .round-button(@color-beta);
+}
+
+// ERROR
```

Comment:
If `.released` and `.error` use the same properties, it can be combined as such:
```css
// Error
.released,
.error {
  .round-button(@color-system-error);
}
```